Sources/Logto/Utilities/LogtoUtilities.swift

```diff
@@ -18,8 +18,8 @@ public enum LogtoUtilities {
 
     static let reservedScopes = ["openid", "offline_access"]
 
-    public static func withReservedScopes(_ scopes: ValueOrArray<String>?) -> [String] {
-        Array(Set((scopes?.inArray ?? []) + reservedScopes)).sorted()
+    public static func withReservedScopes(_ scopes: [String]) -> [String] {
+        Array(Set(scopes + reservedScopes)).sorted()
```

Comment:
Based on our discussion yesterday, the sorting seems to be unnecessary for now

Reply:
yes added LOG-1456

Reply:
ok